docs/build.md

```diff
@@ -0,0 +1,13 @@
+## Building images related to Kubernetes and Kind.
+
+#### Building Kind-node images:
+Kind node images are built with the version of kubernetes, it can be done by triggering the workflow
+under the `Docker` workflow. As an input parameter, one needs to set the version of `Kubernetes` before
```

Comment:
Is there any official K8s link we can post here so one can refer to available Kubernetes versions they can use from?

Reply:
Linked!